pysradb/basedb.py

```diff
@@ -24,7 +25,9 @@ def __init__(self, sqlite_file):
 
     def open(self):
         """Open sqlite connection."""
-        self.db = sqlite3.connect(self.sqlite_file)
+        # Originally sqlite3.connect(self.sqlite_file)
+        self.sqlite_file = self.sqlite_file.replace('?','')
```

Comment:
Is there a need to replace '?'?

Reply:
The '?' character was causing the input to be truncated after '?' in uri syntax. But now since you have clarified that user does not provide a wrong path, I'll remove this check.